src/services/api.ts

```diff
@@ -35,15 +35,12 @@ import { ApiService } from './ApiService';
    *
    * */
 const VERSION = 'v1';
+const prodEndpoint = 'https://api.withampersand.com';
 
 function getApiEndpoint(): string {
-  const prodEndpoint = 'https://api.withampersand.com';
+  const ENV_SERVER = process.env.REACT_APP_AMP_SERVER || 'prod';
```

Comment:
If `process` is undefined, won't this error out?

Reply:
sorry, yeah i forgot to remove review since realized this will still be an issue offline. updated with try catch block.